src/System.Windows.Forms/src/System/Windows/Forms/DataGridView.ToolTip.cs

```diff
@@ -41,12 +39,13 @@ public void Activate(bool activate)
                     };
                 }
 
-                if (activate)
+                if (activate && ToolTip is not null)
                 {
                     ToolTip.Active = true;
                     ToolTip.Show(_dataGridView.ToolTipPrivate, _dataGridView);
                 }
-                else if (ToolTip is not null)
+
+                if (ToolTip is not null)
```

Comment:
I was more thinking about deleting the if clause on lines 32 to 39, and replace this one with
```suggestion
                if (activate)
                {
                    // Create the tool tip handle on demand.
                    ToolTip ??= new ToolTip
                    {
                        ShowAlways = true,
                        InitialDelay = 0,
                        UseFading = false,
                        UseAnimation = false,
                        AutoPopDelay = 0
                    };
                    
                    ToolTip.Active = true;
                    ToolTip.Show(_dataGridView.ToolTipPrivate, _dataGridView);
                }
                else if (ToolTip is not null)
```